apps/client/src/module/common/controlled-input.tsx

```diff
@@ -28,32 +48,46 @@ export function ControlledTextInput<TFieldValues extends object>({
   });
 
   return (
-    <label className="space-y-1">
-      <span className="text-[13px]">{label}</span>
-      <div className="relative flex items-stretch">
-        {!!leftElement && (
-          <span className="bg-primary-100 text-primary-300 p-2 transition-colors duration-300 dark:bg-neutral-700">
-            {leftElement}
+    <label className={inputVariants({ display })}>
+      <span className="text-[13px]">
+        {label}
+        {!!tooltips && !display && (
+          <span className="text-accent-400 hidden group-focus-within:inline">
+            {tooltips}
           </span>
         )}
-        <input
-          className="focus:shadow-accent-400 dark:focus:shadow-accent-400 shadow-primary-100 dark:shadow-primary-700 shadow-inset flex-1 bg-transparent px-2 py-1 text-xs transition-[box-shadow] duration-300 placeholder:text-xs focus:outline-none"
-          placeholder={label ? label : props.placeholder}
-          {...props}
-          {...field}
-        />
-        {!!rightElement && rightElement}
-
-        {error?.message && (
-          <span
-            className={clsx(
-              "text-error-900 bg-error-300 absolute bottom-[calc(100%+4px)] right-0 rounded-md px-2 py-1 text-[10px]",
-              "after:bg-error-300 after:absolute after:-bottom-1 after:right-4 after:h-2 after:w-2 after:rotate-45",
-            )}
-          >
-            {error.message}
+      </span>
+      <div className={clsx(!!tooltips && "flex flex-col")}>
+        {!!tooltips && !!display && (
```

Comment:
feels a little weird that we are showing/hiding tooltips when it's the label that's moving.... but it's working now and I have no complaints!